src/pip/_internal/wheel.py

```diff
@@ -928,10 +928,8 @@ def _base_setup_args(self, req):
         # isolating. Currently, it breaks Python in virtualenvs, because it
         # relies on site.py to find parts of the standard library outside the
         # virtualenv.
-        return [
-            sys.executable, '-u', '-c',
-            SETUPTOOLS_SHIM.format(req.setup_py_path)
-        ] + list(self.global_options)
+        return make_setuptools_shim_args(
+            req.setup_py_path, unbuffered=True) + list(self.global_options)
```

Comment:
Nit: since this has an awkward break, you could do this on its own line:

```python
args = make_setuptools_shim_args(req.setup_py_path, unbuffered=True)
```